### PR TITLE
Integrate gutenberg-mobile release 1.81.0

### PR DESCRIPTION
**Pull request by derekblank**
## Description
This PR incorporates the 1.81.0 release of gutenberg-mobile.
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/5070

Release Submission Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@
 20.5
 -----
 * [*] Use larger thumbnail previews for recommended themes during site creation [https://github.com/wordpress-mobile/WordPress-Android/pull/16848]
+* [***] Block Editor: List block: Adds support for V2 behind a feature flag [https://github.com/WordPress/gutenberg/pull/42702]
```
Comment (geriux) [inline RELEASE-NOTES.txt]: Since this is being released under a feature flag that won't be enabled until mid September, let's add `internal`.

Comment (derekblank) [reply RELEASE-NOTES.txt]: @geriux Good idea, I'll update - do you think the priority level is correct?

Comment (geriux) [reply RELEASE-NOTES.txt]: Yeah I think that's a proper one 👍

**Review by geriux — APPROVED** [review]
LGTM! Thanks for wrangling this release 🚀

**Pull request merged**